app/workers/delete_object_hierarchy_worker.rb

```diff
@@ -74,7 +74,7 @@ def batch_callbacks(batch)
     end
   end
 
-  def delete_associations(object)
+  def delete_associations
```

Comment:
because `object` is [an instance variable](https://github.com/3scale/porta/blob/de3b4578961b7b1c6ccf8cf3f315a84943aec7ef/app/workers/delete_object_hierarchy_worker.rb#L51).